doc/api/util.md

```diff
@@ -126,6 +126,10 @@ util.format(1, 2, 3); // '1 2 3'
 
 ## util.inherits(constructor, superConstructor)
 
+_Note: usage of `util.inherits()` is discouraged. Please use ES6 `class` and
+`extends` to get language level inheritance support, including an unbroken
+inheritance chain._
```

Comment:
Generally I would agree, if the purpose here wouldn't be to discourage them. In that sense it even support the purpose "back-off here". We could add the instanceof / prototype of checks:

```js
class MyError extends Error {}
Error.isPrototypeOf(MyError)
const myerr = new MyError()
myerr instanceof MyError
myerr instanceof Error

function MyError () {}
util.inherits(MyError, Error)
Error.isPrototypeOf(MyError)
const myerr = new MyError()
myerr instanceof MyError
myerr instanceof Error

```